app/src/main/java/com/infomaniak/mail/data/cache/mailboxContent/MessageController.kt

```diff
@@ -115,7 +115,8 @@ class MessageController @Inject constructor(private val mailboxContentRealm: Rea
                     ThreadFilter.UNSEEN -> add("${Message::isSeen.name} == false")
                     ThreadFilter.STARRED -> add("${Message::isFavorite.name} == true")
                     ThreadFilter.FOLDER -> add("${Message::folderId.name} == '$folderId'")
-                    ThreadFilter.ATTACHMENTS -> add("${Message::hasAttachments.name} == true")
+                    ThreadFilter.ATTACHMENTS ->
+                        add("(${Message::hasAttachments.name} == true || ${Message::swissTransferUuid.name} != nil)")
```

Comment:
We should only have 1 field to define if a Message has an Attachment.
The fact that we need to check 2 fields WILL cause issues / oversights in the future.